README.md

```diff
@@ -1,6 +1,76 @@
+    * Create [IoT Hub](https://ms.portal.azure.com/#create/Microsoft.IotHub) and create [DPS](https://ms.portal.azure.com/#create/Microsoft.IoTDeviceProvisioning)
+    * Link IoT Hub to the [DPS scope](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision)
+2. Go to DPS resource and create [individual enrollment](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-symm-key#create-a-device-enrollment-entry-in-the-portal). 
+    * Go to DPS - Manage enrollments - Add individual enrollment
+    * Make sure to use Symmetric Key for attestation type and IoT Edge device == true is selected (default is false)
+    * Retrieve the following information from the DPS resource page
+        * 'Registration ID' (Recommendation: Use the same ID as Device ID for Hub)
+        * ID Scope available in [Overview menu](https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-symm-key#run-the-provisioning-code-for-the-device)
+        * Primary SAS Key from individual enrollment menu
+3. On a device or VM, run the command below. Copy and paste values from above into the script arguments below.
+
+```Command arguments
+wget https://github.com/Azure/iot-edge-config/releases/latest/download/azure-iot-edge-installer.sh -O azure-iot-edge-installer.sh \
+&& chmod +x azure-iot-edge-installer.sh \
+&& sudo -H ./azure-iot-edge-installer.sh -s <IDScope> -r <RegistrationID> -k <Symmetric Key> \
+&& rm -rf azure-iot-edge-installer.sh
+```
+
+Upon completion of the script, go to IoT Hub - IoT Edge page and check the device is created and check system modules ($edgeAgent and $edgeHub) are created. Note that $edgeHub module will only get deployed when the first custom edge module is deployed.
+
+### Test with Azure IoT Central
+1. Create a new IoT Central App; Go to [IoT Central](https://apps.azureiotcentral.com/) and click <+ Build> button. Select any app template or create custom app.
+2. Create a new device by clicking <+ New> button
+3. In 'All devices' view, click on the created devie and click Connect button. Dialoge with ID Scope, Device ID and Primary key will be displayed. Make sure to use authentication type = Shared access signature (SAS)
+4. On a device or VM, run the command below. Copy and paste values from above dialog into the script arguments below.
+
+```Command arguments
+wget https://github.com/Azure/iot-edge-config/releases/latest/download/azure-iot-edge-installer.sh -O azure-iot-edge-installer.sh \
+&& chmod +x azure-iot-edge-installer.sh \
+&& sudo -H ./azure-iot-edge-installer.sh -s <IDScope> -r <RegistrationID> -k <Symmetric Key> \
+&& rm -rf azure-iot-edge-installer.sh
+
+```
+Upon completion of the script, go to All devices view and Device status should be changed from registered to provisioned.
+
+### Uninstalling IoT Edge runtime and Moby-Engine
+You can uninstall both IoT Edge runtime and Moby-Engine with following commands
+
+- Uninstall Edge runtime
```

Comment:
These steps should be run in sequence so we should consider enumerating them (1., 2., etc)

Reply:
Thanks. I changed to the numbers instead of bullets